web/src/pages/news/podcasts/index.page.test.tsx

```diff
@@ -0,0 +1,348 @@
+				query: { page: mockConfig.query.upperOutOfBoundPagination },
+				sbParams: {
+					starts_with: mockConfig.startsWith,
+					per_page: mockConfig.resultsPerPage,
+				},
+			});
+		});
+
+		it("should return notFound if validateRouteParams returns notFound", async () => {
+			validateRouteParamsSpy.mockResolvedValue({ notFound: true });
+
+			const result = await getServerSideProps({
+				query: { page: mockConfig.query.upperOutOfBoundPagination },
+			} as unknown as GetServerSidePropsContext<ParsedUrlQuery>);
+
+			expect(result).toEqual({ notFound: true });
+		});
+
+		it("should return redirect if validateRouteParams returns redirect", async () => {
+			const mockRedirect = {
+				redirect: { destination: "/news/articles", permanent: false },
+			};
+
+			validateRouteParamsSpy.mockResolvedValue(mockRedirect);
+
+			const result = await getServerSideProps({
+				query: { page: mockConfig.query.upperOutOfBoundPagination },
+			} as unknown as GetServerSidePropsContext<ParsedUrlQuery>);
+
+			expect(result).toEqual(mockRedirect);
+		});
+		it("should return error if validateRouteParams returns error", async () => {
+			const mockErrorMessage =
+				"<<<< There was an error fetching stories. Please try again later. >>>>> ";
+			const mockCatchError = new Error(mockErrorMessage);
+
+			validateRouteParamsSpy.mockRejectedValue(mockCatchError);
+
+			const result = await getServerSideProps({
+				query: { page: mockConfig.query.upperOutOfBoundPagination },
+			} as unknown as GetServerSidePropsContext<ParsedUrlQuery>);
+
+			expect(result).toEqual({ props: { error: mockErrorMessage } });
+		});
+
+		it("should return a general error message if there is a throw that is not an Error object", async () => {
+			const mockCatchError = "some string error";
+
+			validateRouteParamsSpy.mockRejectedValue(mockCatchError);
+
+			const result = await getServerSideProps({
+				query: { page: mockConfig.query.upperOutOfBoundPagination },
+			} as unknown as GetServerSidePropsContext<ParsedUrlQuery>);
+
+			expect(result).toEqual({
+				props: {
+					error:
+						"Oops! Something went wrong and we're working to fix it. Please try again later.",
+				},
+			});
+		});
+
+		it("should return a correct props object if validateRouteParams returns success", async () => {
+			const mockValidatedRouteParamsSuccessResponse = {
+				featuredStory: mockStories[0],
+				stories: mockStories.slice(1),
+				total: MockStoryblokSuccessResponse.total,
+				currentPage: mockConfig.currentPage,
+				perPage: MockStoryblokSuccessResponse.perPage,
+			};
+			// MockStoryblokSuccessResponse
+			validateRouteParamsSpy.mockResolvedValue(
+				mockValidatedRouteParamsSuccessResponse
+			);
+
+			const result = await getServerSideProps({
+				query: { page: mockConfig.query.pageAt1 },
+			} as unknown as GetServerSidePropsContext<ParsedUrlQuery>);
+
+			expect(result).toEqual({
+				props: {
+					//TODO check functionality around featuredStory merging - does this suffice?
```

Comment:
I'll stop commenting on the TODOs - I think the important thing is to go through each one and ask:

- Are you happy with how the code is working/what it's doing, so can you delete the comment?
- If work still needs doing here, would it be better to add it to the tech debt spreadsheet so that it doesn't get lost?

Don't want to be too dogmatic about it and there's certainly a place for the odd TODO here and there, just want to make sure that we're probably logging or satisfying any concerns as there are quite a few TODOs in this PR